keras_preprocessing/text.py

```diff
@@ -514,4 +514,4 @@ def tokenizer_from_json(json_string):
     tokenizer.word_index = word_index
     tokenizer.index_word = index_word
 
-    return tokenizer
```

Comment:
Please remove any changes to this file.

Reply:
sure thing (my IDE auto-trims trailing whitespace on file save but I'll restore it)

Reply:
that's the reason the CI is faailing https://travis-ci.org/keras-team/keras-preprocessing/jobs/617265242?utm_medium=notification&utm_source=github_status

Reply:
ah, I see! I'll use vim instead